<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="my_HP">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="enemy_HP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="enemy_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="my_pic">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fight">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>싸운다</string>
    </property>
   </widget>
   <widget class="QPushButton" name="potion">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>포션</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>도망친다</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enemy_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="my_name">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="battle_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
